Shared/Samples/Apply mosaic rule to rasters/ApplyMosaicRuleToRastersView.swift

```diff
@@ -0,0 +1,211 @@
+                        }
+                        .onChange(of: model.ruleSelection) { ruleSelection in
+                            Task {
+                                model.updateMosiacRule(with: ruleSelection.rule)
+                                await mapProxy.setViewpoint(
+                                    Viewpoint(
+                                        center: model.getCenter(),
+                                        scale: 25000.0
+                                    )
+                                )
+                            }
+                        }
+                        .pickerStyle(.automatic)
+                    }
+                }
+        }
+        .errorAlert(presentingError: $error)
+    }
+}
+
+private enum RuleSelection: CaseIterable, Equatable {
+    static var allCases: [RuleSelection] {
+        return [.objectID, .northWest, .center, .byAttribute, .lockRaster]
+    }
+    
+    case objectID, northWest, center, byAttribute, lockRaster
+    
+    @available(*, unavailable)
+    case all
+    
+    var label: String {
+        switch self {
+        case .objectID:
+            return "Object ID"
+        case .northWest:
+            return "North West"
+        case .center:
+            return "Center"
+        case .byAttribute:
+            return "By Attribute"
+        case .lockRaster:
+            return "Lock Raster"
+        }
+    }
+    
+    var rule: MosaicRule {
+        switch self {
+        case .objectID:
+            // A default mosaic rule object, with mosaic method as objectID which
+            // functionally is the same as the none rule in earlier versions.
+            let objectIDRule = MosaicRule()
+            objectIDRule.mosaicMethod = .objectID
+            return objectIDRule
+        case .northWest:
+            // A mosaic rule object with northwest method.
+            let northWestRule = MosaicRule()
+            northWestRule.mosaicMethod = .northwest
+            northWestRule.mosaicOperation = .first
+            return northWestRule
+        case .center:
+            // A mosaic rule object with center method and blend operation.
+            let centerRule = MosaicRule()
+            centerRule.mosaicMethod = .center
+            centerRule.mosaicOperation = .blend
+            return centerRule
+        case .byAttribute:
+            // A mosaic rule object with byAttribute method and sort on "OBJECTID" field of the service.
+            let byAttributeRule = MosaicRule()
+            byAttributeRule.mosaicMethod = .attribute
+            byAttributeRule.sortField = "OBJECTID"
+            return byAttributeRule
+        case .lockRaster:
+            // A mosaic rule object with lockRaster method and locks 3 image rasters.
+            let lockRasterRule = MosaicRule()
+            lockRasterRule.mosaicMethod = .lockRaster
+            lockRasterRule.addLockRasterIDs([1, 7, 12])
+            return lockRasterRule
+        }
+    }
```

Comment:
You can also create and return the `MosiacRule` once to reduce redundant code. This also probably could use a doc comment.

```swift
var rule: MosaicRule {
    let mosaicRule = MosaicRule()
    
    switch self {
    case .objectID:
        mosaicRule.mosaicMethod = .objectID
    case .northWest:
        mosaicRule.mosaicMethod = .northwest
        mosaicRule.mosaicOperation = .first
        ...
    }
    return mosaicRule
}
```